app/javascript/packages/document-capture/components/in-person-location-step.tsx

```diff
@@ -30,9 +30,63 @@ interface FormattedLocation {
   weekdayHours: string;
 }
 
+interface RequestOptions {
+  /**
+   * Whether to send the request as a JSON request. Defaults to inclusion.
+   */
+  json: boolean;
+
+  /**
+   * Whether to include CSRF token in the request. Defaults to inclusion.
+   */
+  csrf: boolean;
+
+  /**
+   * Optional. HTTP verb used. If excluded, will fetch as a GET.
+   */
+  method: string;
```

Comment:
supernit: For some reason, the default language is a little confusing to me. How about these:

(also if `method` can be left out, or any of them, should we make them optional properties?)

```suggestion
interface RequestOptions {
  /**
   * Whether to send the request as a JSON request. Defaults to true.
   */
  json?: boolean;

  /**
   * Whether to include CSRF token in the request. Defaults to true.
   */
  csrf?: boolean;

  /**
   * Optional. HTTP verb used. Defaults to GET.
   */
  method?: string;
```